pkg/util/consts.go

```diff
@@ -38,3 +50,15 @@ const (
 	HCONSWebhookPath            = "/mutate-ns-hco-kubevirt-io"
 	WebhookPort                 = 4343
 )
+
+type AppComponent string
+
+const (
+	ComputeComponent    AppComponent = "compute"
+	StorageComponent                 = "storage"
+	ImportComponent                  = "import"
```

Comment:
Don't know if "import" merits a component name of its own (i.e., if its significant enough for the user?). Maybe this can fit as part of "compute" component? Or maybe "migration" is a better name?

Reply:
It was suggested by @fabiand  at the beginning of the discussions.